### PR TITLE
Fix typos in README.fr-FR.md

### PR DESCRIPTION
**Pull request by Lenophie**
This PR is a further improvement of #146

### CODE REVIEW

**Comment by flawyte** [comment]
+1 for rephrasing some parts.
However, in the original README « **don't send** » (« n'envoyez pas » here) is in bold and there's probably a good reason to that, so I think it'd better not to remove that formatting.

**Comment by Lenophie** [comment]
Fixed, it was an oversight on my part

File: README.fr-FR.md [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
 
 # Contribuez !
-Les fichiers sources de ce dépôt sont mis à disposition pour référence historique et ne seront pas modifiés, donc s'il vous plait **n'envoyer pas** de Pull Requests suggérant des modifications aux fichiers sources, mais forkez ce dépôt et expérimentez 😊.  
+Les fichiers sources de ce dépôt sont mis à disposition pour référence historique et ne seront pas modifiés, donc s'il-vous-plaît **n'envoyez pas** de Pull Requests suggérant des modifications aux fichiers sources, mais forkez ce dépôt et expérimentez 😊.  
 
-Si, toutefois, vous souhaitiez soumettre du contenu additionel ou des modifications à des fichiers autres que les sources, (par exemple ce README), soumettez via une PR, nous ferons une revue et la prendrons en compte.
+Si, toutefois, vous souhaiteriez soumettre du contenu additionel ou des modifications à des fichiers autres que les sources (par exemple ce README), soumettez vos changements via une PR, nous en ferons une revue et la prendrons en compte.
```
Comment (Pandraghon) [inline README.fr-FR.md]:
```suggestion
Si, toutefois, vous souhaiteriez soumettre du contenu additionnel ou des modifications à des fichiers autres que les sources (par exemple ce README), soumettez vos changements via une PR, nous en ferons une revue et la prendrons en compte.
```

Comment (flawyte) [reply README.fr-FR.md]: Much better!

**Pull request closed without merging**